packages/api/amplify_api/lib/src/graphql/ws/websocket_connection.dart

```diff
@@ -0,0 +1,186 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:amplify_api/src/decorators/authorize_websocket_message.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'package:async/async.dart';
+import 'package:web_socket_channel/web_socket_channel.dart';
+
+import 'websocket_message.dart';
+import 'websocket_message_stream_transformer.dart';
+
+/// {@template websocket_connection}
```

Comment:
Template names should be globally unique

```suggestion
/// {@template amplify_api.ws.websocket_connection}
```